### PR TITLE
Smith dev

### PR DESCRIPTION
**Pull request by SmithcS**
Has not been tested cross browser (chrome only)

### CODE REVIEW

File: website/css/main-compiled.css [hunk 1]
```diff
@@ -11,6 +11,18 @@
   background-color: #AACACC;
```
Comment (sushshring) [inline website/css/main-compiled.css]: Can you gitignore this file? No point committing compiled code.

File: website/index.html [hunk 2]
```diff
@@ -132,68 +133,90 @@
...
                 </div>
-              </a>
+            </div>
+          </div>
+
+          <!-- Mountain vista graphics -->
+          <div class="col-md-7">
+            <div class="row header-graphic">
+              <img src="assets/images/flat-mountain-graphic-snow.svg" class="img-fluid mountain">
+              <!-- Each cloud img needs its own div to keep their position correct on resize
```
Comment (sushshring) [inline website/index.html]: nice.

File: website/index.html [hunk 3]
```diff
@@ -132,68 +133,90 @@
...
-        <img class="section-separator-background" src="assets/images/border1_wTransparency.svg">
-      </div>
     </header>
+
     <section id="about">
       <div class="container">
+
         <div class="row">
-          <div class="section-header col-xs-12">
+          <div class="section-header col-md-12">
```
Comment (sushshring) [inline website/index.html]: What happens on smaller devices? Shouldn't this span the width for all?

File: website/index.html [hunk 4]
```diff
@@ -132,68 +133,90 @@
...
           </div>
-          <p>
-            This October 6 - 7th, join us and over 700 students from across the nation for 24 hours
-            of creation, innovation, & fun. From seasoned coding veterans to first-time hackers from
-            any major or field, all are welcome to join. Discover your ability to create change by
-            developing technology to solve real-world problems, working with industry leaders, and
-            collaborating with your peers.
-          </p>
-          <p>Of course, we also offer free food, swag, prizes, and much more for your enjoyment!</p>
+          <div class="col-md-12 about-text">
```
Comment (sushshring) [inline website/index.html]: Same as above.

File: website/index.html (hunk 3 above)
Comment (SmithcS) [reply website/index.html]: Fixed in latest commits - just used generic col-12

File: website/css/main-compiled.css (hunk 1 above)
Comment (SmithcS) [reply website/css/main-compiled.css]: Yep, just added it

**Comment by SmithcS** [comment]
Done with updates for now. Will consider this release v1.0.1
Additional changes I make will be toward v1.0.2 or whatever we'd like to call it

**Comment by sushshring** [comment]
One of the backlog items was the lack of social media icons. Is this taken care of here?

**Comment by sushshring** [comment]
Also: `Change the Title meta tag on the landing page`. Can this be included with this PR?

**Comment by SmithcS** [comment]
Yes & yes, just looked and all the icons are linking to the right places, and the meta tag is now accurate  👍

**Comment by Ryguy11o** [comment]
Also @SmithcS i dont think the blue highlights in the About section look good. I would suggest removing those and just replacing them with bold text

**Comment by sushshring** [comment]
Nice responsiveness!

**Comment by SmithcS** [comment]
@Ryguy11o Done, might be worth just going back to normal text
@sushshring Thanks 🤙

**Comment by sushshring** [comment]
PLS Squash and merge to Master

File: website/scss/main.scss [hunk 5]
```diff
@@ -200,44 +280,106 @@ header {
...
+        animation-delay: -2s;
+      }
+    }
+    .mountain {
+      position: relative;
+      top: 0px;
+      margin: auto;
+      width: 100%;
+      padding-right: 5%;
+      z-index: 200;
```
[the remark this answers is not in the document]
Comment (SmithcS) [reply website/scss/main.scss]: Just following convention I've seen other places - I think its mean to allow for control with z-index selection (like grouping a bunch of objects at level 100, and then choosing how they layers with 101, 102, etc)

**Pull request merged**